pyuvsim/tests/test_simsetup.py

```diff
@@ -704,125 +807,112 @@ def test_freq_parser_errors(freq_dict, msg):
         pyuvsim.parse_frequency_params(freq_dict)
 
 
-def test_time_parser():
+@pytest.mark.parametrize(
+    "time_keys",
+    [
+        ['start_time', 'end_time', 'duration_hours', 'Ntimes'],
+        ['start_time', 'end_time', 'integration_time'],
+        ['start_time', 'integration_time', 'Ntimes', 'duration_hours'],
+        ['end_time', 'integration_time', 'Ntimes', 'duration_hours'],
+        ['start_time', 'integration_time', 'Ntimes'],
+        ['end_time', 'integration_time', 'Ntimes'],
+        ['start_time', 'duration_hours', 'Ntimes'],
+        ['end_time', 'duration_hours', 'Ntimes'],
+        ['start_time', 'duration_hours', 'integration_time'],
+        ['end_time', 'duration_hours', 'integration_time'],
+        ['time_array'],
```

Comment:
It's been a bit since I've looked at the parse time params, want to make sure that these 11 cases are all we need and not all 32 from choosing each of the 5 options here.